deploy/requirements.yml

```diff
@@ -2,4 +2,4 @@
 # Install roles with
```

Comment:
I think we should vendor and maintain all our roles internally to keep consistency, it'll mean that anyone can pull down our repo and it should work. Otherwise you tend to get role drift (ansible has no way to pick role versions). Keeps things simple too.